www.chuyu.me/es.xml

```diff
@@ -13,9 +13,9 @@
   <String Src="%AboutDism++%" Dst="Acerca de Dism++"/>
   <String Src="%AboutDism++Des%" Dst="Dism++ puede considerarse como una versión de Dism con GUI, pero basada en servicios de componentes (CBS) de bajo nivel en lugar de DISM API o DISM Core API."/>
   <String Src="%OpenSourceStr%" Dst="¡El código abierto no es sólo una virtud, sino también un símbolo de fortaleza!. &lt;span onclick=&quot;window.location.href='https://github.com/Chuyu-Team'&quot; style=&quot;color:blue;cursor:pointer&quot;>Clic aquí&lt;/span> para ver el equipo de Chuyu en GitHub."/>
-  <String Src="%SxSClean%" Dst="Limpiar WinSxS"/>
+  <String Src="%SxSClean%" Dst="Limpieza de WinSxS"/>

```

Comment:
is different when expressing features dism ++, in the page.

In the program Dism++ is correct since it executes actions, but in the page not since they are features that have Dism++.